object_ids:
  - ground

object_types:
  - box

ground: {
  position: {
    x: 0.0,
    y: 0.0,
    z: -0.05,
  },
  dimensions: {
    length: 1.5,
    width: 1.5,
    height: 0.1,
  },
  primitive_type: box,
  orientation_type: up_x,
}

object_orientation_types:
  - up_x